Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM gocd/gocd-agent-docker-dind:v18.7.0
 
-RUN apk --no-cache add python3 openssl ca-certificates
+COPY requirements.txt .
+COPY requirements requirements/
+
+RUN apk --no-cache add python3 openssl ca-certificates openblas-dev gfortran
 RUN apk --update add --virtual build-dependencies python3-dev libffi-dev build-base linux-headers \
-  && pip3 install --no-cache-dir dvc \
+  && pip3 install --no-cache-dir --upgrade pip \
+  && pip install --no-cache-dir -r requirements.txt \
```

Comment:
Shouldn't this be `pip3 install`? I only had the symlink from pip3 to pip after that.. or after upgrading then pip in the PATH is already pip3?

Reply:
`pip3 install --upgrade pip` makes `pip` point to the python3 version by default, which is the desired behavior

Reply:
Cool, in that case, we don't need the `ln` on line 11